### ISSUE TITLE
MongoProperties is not used at all but it is required

### ISSUE THREAD

**Issue by fcanovas1986** [javers/javers#1324]
**Clear description of my expectations versus reality**

Having configured the dependency `javers-spring-boot-starter-mongo` , when I run my application, I have the following error:

```
***************************
APPLICATION FAILED TO START
***************************

Description:

Field mongoProperties in `org.javers.spring.boot.mongo.JaversMongoAutoConfiguration` required a bean of type 'org.springframework.boot.autoconfigure.mongo.MongoProperties' that could not be found.


Action:

Consider defining a bean of type 'org.springframework.boot.autoconfigure.mongo.MongoProperties' in your configuration.
```
That says that `JaversMongoAutoConfiguration` needs the `MongoProperties` bean. I don't use this bean to set mongo settings, I think it should be optional. What's more, I checked that MongoProperties bean is autowired and it is **required** in JaversMongoAutoConfiguration, **but it is not used at all**, so you could even remove it. But if this is not the case for any reason, at least should be Optional.

https://github.com/javers/javers/blob/b83fbd1176c3ea10d7ecfc167b948a14959ccd8a/javers-spring-boot-starter-mongo/src/main/java/org/javers/spring/boot/mongo/JaversMongoAutoConfiguration.java#L63

**Steps To Reproduce**

Configure the  `javers-spring-boot-starter-mongo` and does not declare a MongoProperties bean in your application and run.

**Javers' Version**

7.3.2

**Additional context**

**Comment by bartoszwalacik** [comment]
@fcanovas1986 thanks for reporting, you are right this property seems unused and it's easy to fix. Feel free to contribute a PR with the fix

**Comment by fcanovas1986** [comment]
Hi @bartoszwalacik , could you review the opened PR? it is very little :) .

Thanks!

**Comment by bartoszwalacik** [comment]
fixed released in 7.3.3
